spec/spec_helper.rb

```diff
@@ -169,7 +169,7 @@ module Shell
   config.filter_run_excluding linux_only: true unless linux?
   config.filter_run_excluding aix_only: true unless aix?
   config.filter_run_excluding suse_only: true unless suse?
-  config.filter_run_excluding sles11: true unless sles11?
+  config.filter_run_excluding opensuse15: true unless opensuse15?
```

Comment:
can we just change this to opensuse so we catch 16 as well where this most likely continues to work. If not when we upgrade those testers in buildkite these tests will just disappear.

Reply:
Updated @tas50.